### ISSUE TITLE
curl does not know about c-ares

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#110]
Currently we ship libcurl as part of KDR recording package. The libcurl is a default installation module required by webrec data recording. It has been noticed that libcurl does not use c-ares library:

```
WARNING: Your libcurl doesn't appear to be built with c-ares support.  This
could significantly compromise the performance of programs that use
AnyEvent::Curl::Multi.   You will either need to contact your OS vendor to
obtain an update of libcurl with c-ares support, or compile a local version of
libcurl with c-ares support (e.g. in /usr/local or /opt/local) and ensure
WWW::Curl is linked against it.
```

c-ares is a C library for asynchronous DNS requests (including name resolves)

**Comment by sparvu** [comment]
will be fixed in 1.1 release

**Comment by sparvu** [comment]
done.
